genesis/genesis_test.go

```diff
@@ -118,15 +121,6 @@ func TestValidateConfig(t *testing.T) {
 			}(),
 			expectedErr: errInitialStakeDurationTooLow,
 		},
-		"empty initial staked funds": {
-			networkID: 12345,
-			config: func() *Config {
-				thisConfig := LocalConfig
-				thisConfig.InitialStakedFunds = []ids.ShortID(nil)
-				return &thisConfig
-			}(),
-			expectedErr: errNoInitiallyStakedFunds,
-		},
```

Comment:
Did we mean to remove this? I think this is a different case than the insufficient allocations case

Reply:
definitely not, i'll need to add this back